tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -183,6 +193,14 @@ This section describes how to build and deploy in debug mode.
    in containerd as well. Note that this variable has no effect if
    `SHIM_REDEPLOY_CONFIG=no`.
 
+`make all-confpods` builds a pair of release and debug UVMs and IGVMs by default. For quicker development, `make-uvm-confpods-[release/debug]` targets are available.
+
+```shell
+make uvm-confpods-debug
+make package-confpods
+make deploy-confpods
+```
+
```

Comment:
I wouldn't add a paragraph - rather edit the existing commands in this Debug builds section.

Reply:
Addressed in latest push; # Debug section is no longer specific to confpods and I inserted documentation there.